balancer/weightedtarget/weightedtarget_test.go

```diff
@@ -323,6 +364,11 @@ func (s) TestWeightedTarget(t *testing.T) {
 	if state != connectivity.TransientFailure {
 		t.Fatalf("Empty target update should have triggered a TF state update, got: %v", state)
 	}
+	p = <-cc.NewPickerCh
+	const wantErr = "no targets to pick from"
+	if _, err := p.Pick(balancer.PickInfo{}); !strings.Contains(err.Error(), wantErr) {
```

Comment:
If `err` is nil this will panic.

```suggestion
	if _, err := p.Pick(balancer.PickInfo{}); err == nil || !strings.Contains(err.Error(), wantErr) {
```

Or

```go
	if _, err := p.Pick(balancer.PickInfo{}); !strings.Contains(fmt.Sprint(err), wantErr) {
```

Reply:
Thanks. Done.